### PR TITLE
mockup of dynamic vehicle list

### PR DESCRIPTION
**Pull request by fabianbormann**

### CODE REVIEW

File: components/gn-ui-menu.html [hunk 1]
```diff
@@ -105,6 +104,14 @@
         },
 
         /**
+         * Holds supported vehicles 
+         */
+        vehicles: {
```
Comment (jinankjain) [inline components/gn-ui-menu.html]: @fabianbormann Is it justified to define vehicle as a properties of menu component. As it is totally irrelevant in gn-menu? Could we don't have any other method to do this?

**Comment by fabianbormann** [comment]
Another method could be to add gn-api also to gn-ui-menu but I think that way would be very ugly. What is your decision?

**Comment by fabianbormann** [comment]
It think it's not "totally irrelevant" because we need it in the template iterator, so this must be a array property. I think this is a really short and readable way to pass the vehicle list - but if you have a better decision please share it with me

**Comment by jinankjain** [comment]
@fabianbormann The one which I was trying to do with my pull request. What about that one? Like first we do document.querySelector on gn-api that we are already loading in index.html and using that we call the function getVehicleList and iterate over the list that we obtain.

**Comment by jinankjain** [comment]
Other way could be like appending child element of paper-item in the dom tree for each vehicle in vehicleList

**Pull request merged**